modules/Vtiger/models/Record.php

```diff
@@ -486,7 +486,9 @@ public static function getCleanInstance($moduleName)
 		$instance = new $modelClassName();
 		$instance->setModuleFromInstance($module);
 		$instance->isNew = true;
-		$instance->setData($focus->column_fields)->setModule($moduleName)->setEntity($focus);
+		if (property_exists($focus, 'column_fields')) {
+			$instance->setData($focus->column_fields)->setModule($moduleName)->setEntity($focus);
```

Comment:
Always have to set ->setModule($moduleName)->setEntity($focus);
->setData($focus->column_fields) may be in the condition of the rest of the outside